### PR TITLE
Added another project in mobile dev

### PR DESCRIPTION
**Pull request by SaptarshiSarkar12**
## Changes proposed
Added newpipe project.

### CODE REVIEW

**Comment by SaptarshiSarkar12** [comment]
@kaiwalyakoparkar Please review this PR.

File: Mobile-Development/Contribute/Readme.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 - [mediapipe](https://github.com/google/mediapipe)
 - [mirai](https://github.com/mamoe/mirai)
 - [MMKV](https://github.com/Tencent/MMKV)
+- [NewPipe](https://github.com/TeamNewPipe/New Pipe)
```
Comment (kaiwalyakoparkar) [inline Mobile-Development/Contribute/Readme.md:20]: This link is not working

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Brittle

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]
Link is not working. Please change it to working one please

**Comment by SaptarshiSarkar12** [comment]
@kaiwalyakoparkar I have fixed the link, please review it.

**Comment by SaptarshiSarkar12** [comment]
@kaiwalyakoparkar I think this PR is ready to be merged.

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Pull request merged**